browser-test/src/support/admin_questions.ts

```diff
@@ -728,13 +758,72 @@ export class AdminQuestions {
   }
 
   /** Changes the input field of a multi option answer. */
-  async changeMultiOptionAnswer(index: number, text: string) {
+  async changeMultiOptionAnswer(index: number, optionText: string) {
     await this.page.fill(
-      `:nth-match(#question-settings div.cf-multi-option-question-option, ${index}) input`,
-      text,
+      `:nth-match(#question-settings div.cf-multi-option-question-option, ${
+        index + 1
+      }) .cf-multi-option-input input`,
+      optionText,
     )
   }
 
+  async fillMultiOptionAnswer(index: number, option: QuestionOption) {
+    await this.page.fill(
+      `:nth-match(#question-settings div.cf-multi-option-question-option, ${
```

Comment:
**suggestion:** You can consider extracting this and the other "nth-match" item into a constant, since its used in the next function also.